.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Continuous Integration
 
-on: push
+on: pull_request
```

Comment:
Do we need both `push` and `pull_request` events to trigger CI if we make changes to a PR based on reviews?

Reply:
pull_request events are triggered on `open`, `synchronize` and `reopen` sub events. `synchronize` sub event covers the case we need.